specs/phase1/beacon-chain.md

```diff
@@ -614,106 +358,24 @@ def get_start_shard(state: BeaconState, slot: Slot) -> Shard:
     Return the start shard at ``slot``.
     """
     current_epoch_start_slot = compute_start_slot_at_epoch(get_current_epoch(state))
-    active_shard_count = get_active_shard_count(state)
-    if current_epoch_start_slot == slot:
-        return state.current_epoch_start_shard
-    elif slot > current_epoch_start_slot:
+    shard = state.current_epoch_start_shard
+    if slot > current_epoch_start_slot:
         # Current epoch or the next epoch lookahead
-        shard_delta = get_committee_count_delta(state, start_slot=current_epoch_start_slot, stop_slot=slot)
-        return Shard((state.current_epoch_start_shard + shard_delta) % active_shard_count)
-    else:
+        for _slot in range(current_epoch_start_slot, slot):
```

Comment:
```suggestion
        for _slot in range(current_epoch_start_slot, slot - 1):
```
Otherwise we will have the start shard at `slot + 1`.

Reply:
The last slot in `range(current_epoch_start_slot, slot)` _would be_ `slot-1`; that's how ranges in python work.